.travis.yml

```diff
@@ -1,6 +1,6 @@
 # Travis CI configuration file for psycopg2
 
-dist: trusty
```

Comment:
If moving to bionic you find some old db unavailable, that's fine. Postgres atm only supports from 9.4 on and that's what we'd do.

Reply:
Both bionic and xenial fails during installation of some versions... I investigate...
Seems I can move to xenial with EOL db versions. But still have trouble with install of actual releases.